sycl/include/CL/sycl/detail/spirv.hpp

```diff
@@ -224,11 +224,11 @@ EnableIfGenericBroadcast<T> GroupBroadcast(T x, id<Dimensions> local_id) {
 // Single happens-before means semantics should always apply to all spaces
 // Although consume is unsupported, forwarding to acquire is valid
 template <typename T>
-static inline constexpr typename std::enable_if<
-    std::is_same<T, sycl::ext::oneapi::memory_order>::value ||
-        std::is_same<T, sycl::memory_order>::value,
-    __spv::MemorySemanticsMask::Flag>::type
-getMemorySemanticsMask(T Order) {
+static inline constexpr
+    typename std::enable_if<std::is_same<T, sycl::memory_order>::value ||
+                                std::is_same<T, sycl::memory_order>::value,
```

Comment:
Could this be simplified to `std::enable_if<std::is_same<T, sycl::memory_order>::value>` now?

Reply:
I may have been a bit quick with that one. Good eye!